src/common/mfu_daos.c

```diff
@@ -619,18 +654,22 @@ static int daos_any_error(int rank, bool local_daos_error, int flag_daos_args)
  * Returns 1 on error, 0 on success.
  */
 static int cont_get_props(daos_handle_t coh, daos_prop_t** _props,
-                          bool get_oid, bool get_label)
+                          bool get_oid, bool get_label, bool get_roots)
 {
     int             rc;
     daos_prop_t*    props = NULL;
     daos_prop_t*    prop_acl = NULL;
     daos_prop_t*    props_merged = NULL;
-    uint32_t        num_props = 14;
+    uint32_t        num_props = 15;
 
     if (get_oid) {
         num_props++;
     }
 
+    if (get_roots) {
+        num_props++;
+    }
+
```

Comment:
Maybe move this block to come just after the ``if (get_label) num_props++;`` to keep with the same order of ``get_oid, get_label, get_roots`` that is used when setting values below.